package.json

```diff
@@ -122,8 +123,8 @@
     "test:current": "node node_modules/.bin/jest --watch",
     "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
-    "deploy-storybook": "storybook-to-ghpages",
+    "build-storybook": "cross-env NODE_OPTIONS=--max_old_space_size=8192 build-storybook",
+    "deploy-storybook": "cross-env NODE_OPTIONS=--max_old_space_size=8192 storybook-to-ghpages",
```

Comment:
Does the deploy actually need it as well? It should just do `push to git` afaik

Reply:
yes, the deploy script builds the static storybook, so it needed as well